### DISCUSSION TITLE
Filter invoices by Lightning Address

### DISCUSSION THREAD

**Discussion by vindard** [btcpayserver/btcpayserver#5751] [Ideas / Feature Requests]
## Description

I currently am able to filter my invoices by the Point-of-Sale checkout page I use, but there's no way to filter out the lightning address received payments in the `Invoices` list.

**Comment by dennisreimann** [comment]
I once made a proof of concept for something that sounds like it might solve your problem: #4422

Or is your use case more about filtering and having the option to search for invoices that were paid to a Lightning Address?
